tests/api/token/temporary.test.js

```diff
@@ -0,0 +1,86 @@
+import { createMocks } from "node-mocks-http";
+import temporary from "../../../pages/api/token/temporary";
+import executeQuery from "@lib/integration/queryManager";
+import jwt from "jsonwebtoken";
+
+jest.mock("next-auth/client");
+jest.mock("@lib/integration/queryManager");
+
+jest.mock("@lib/integration/dbConnector", () => {
+  const mockClient = {
+    connect: jest.fn(),
+    query: jest.fn(),
+    end: jest.fn(),
+  };
+  return jest.fn(() => mockClient);
+});
+
+describe("TemporaryBearerToken tests", () => {
+  beforeAll(() => {
+    process.env.TOKEN_SECRET = "some_secret_some_secret_some_secret_some_secret";
+    process.env.TOKEN_SECRET_WRONG = "wrong_secret_wrong_secret_wrong_secret_wrong_secret";
+  });
+  afterAll(() => {
+    delete process.env.TOKEN_SECRET;
+    delete process.env.TOKEN_SECRET_WRONG;
+  });
+
+  it("works as expected", async () => {
```

Comment:
The test description should be explicite. I can't tell what is expected only by reading its description.